build.sbt

```diff
@@ -122,7 +122,7 @@ lazy val `support-workers` = (project in file("support-workers"))
     `acquisition-event-producer`,
     `module-bigquery`,
     `supporter-product-data-dynamo`
-  ).aggregate(`support-services`, `support-models`, `support-config`, `support-internationalisation`, `stripe-intent`, `acquisition-event-producer`)
+  ).aggregate(`support-services`, `support-models`, `support-config`, `support-internationalisation`, `stripe-intent`, `acquisition-event-producer`, `supporter-product-data-dynamo`)
```

Comment:
always a tricky one haaving to do dependsOn for the classpath and aggregate so that the tests run